stubs/luigi/__init__.pyi

```diff
@@ -0,0 +1,20 @@
+# Stubs for luigi (Python 3.5)
```

Comment:
I assume these .pyi files are not needed?  IDE operator error?

Reply:
no, the .pyi files are needed; they're [type hints](https://docs.python.org/3/library/typing.html)